webapp/channels/src/components/post_view/channel_intro_message/channel_intro_message.tsx

```diff
@@ -91,7 +95,47 @@ export default class ChannelIntroMessage extends React.PureComponent<Props> {
     }
 }
 
-function createGMIntroMessage(channel: Channel, centeredIntro: string, profiles: UserProfileType[], currentUserId: string) {
+const gmIntroMessages = defineMessages({
+    muted: {id: 'intro_messages.GM.muted', defaultMessage: 'This group message is currently <b>muted</b>, so you will not be notified.'},
+    [NotificationLevel.ALL]: {id: 'intro_messages.GM.all', defaultMessage: 'You\'ll be notified <b>for all activity</b> in this group message.'},
+    [NotificationLevel.DEFAULT]: {id: 'intro_messages.GM.all', defaultMessage: 'You\'ll be notified <b>for all activity</b> in this group message.'},
+    [NotificationLevel.MENTION]: {id: 'intro_messages.GM.mention', defaultMessage: 'You have selected to be notified <b>only when mentioned</b> in this group message.'},
+    [NotificationLevel.NONE]: {id: 'intro_messages.GM.none', defaultMessage: 'You have selected to <b>never</b> be notified in this group message.'},
+});
```

Comment:
Question : Was the mmjstool not able to identify the translations that we had to do this here?

Reply:
The alternative would have been either define them with the `t` function (which we want to avoid) or having one `<FormattedMessage />` component for each string. That is why I used `defineMessages`.

Are you envisioning any other way I may be missing?

Reply:
No i was under impression that the tools wasn't able to extract, because of its limitation. For example, the tool cannot extract if translations are in ternary operator
```
const a = condition ? formatMessage(..) : formatMessage(..);
```
The above doesn't gets extracted last time i checked